.travis.yml

```diff
@@ -182,6 +182,8 @@ jobs:
       env: [ ARCH=x86_64, BUILD_TYPE=Debug, SANITIZER=address ]
     - <<: *macos1015_xcode11_5
       env: [ ARCH=x86_64, BUILD_TYPE=Release ]
+    - <<: *macos1015_xcode11_5
```

Comment:
It seems like we forgot to remove `.travis.yml`, it hasn't been maintained for quite some time. You can throw it away, or I'll do it ...